config/controller.yaml

```diff
@@ -71,9 +71,6 @@ spec:
           "-sidecarlogresults-image", "ko://github.com/tektoncd/pipeline/cmd/sidecarlogresults",
           "-workingdirinit-image", "ko://github.com/tektoncd/pipeline/cmd/workingdirinit",
 
-          # This is gcr.io/google.com/cloudsdktool/cloud-sdk:302.0.0-slim
-          "-gsutil-image", "gcr.io/google.com/cloudsdktool/cloud-sdk@sha256:27b2c22bf259d9bc1a291e99c63791ba0c27a04d2db0a43241ba0f1f20f4067f",
```

Comment:
We can also remove https://github.com/tektoncd/pipeline/blob/main/config/300-resource.yaml as well right?

Reply:
Thanks, removed:)